main.nf

```diff
@@ -77,6 +77,16 @@ process gather_fastqs {
     file "bactopia.versions" optional true
     file "multiple-read-sets-merged.txt" optional true
 
+    stub:
+    """
+    mkdir fastqs
+    mkdir extra
+    touch /*-error.txt
+    touch /fastqs/${sample}*.fastq.gz
+    touch /extra/*.gz
+    """
```

Comment:
@Mxrcon , this is a good step forward.

However, the rest of the changes so far are simply noise. Please avoid creating stylistic changes in PRs.

Reply:
Everything which is simply stylistic change needs to be removed from this PR.

Reply:
Ok! I'll avoid every stylistic modification in the future